publications.rkt

```diff
@@ -67,6 +67,51 @@
 
 @(define publications
    (list
+    (publication "Realistic Realizability: Specifying ABIs You Can Count On"
+                 "Andrew Wagner, Zachary Eisbach, and Amal Ahmed"
+                 "Object-Oriented Programming Systems, Languages, and Applications (OOPSLA)"
+                 2024
+                 "https://doi.org/10.1145/3689755")
```

Comment:
this doi 404's for me 🤔

Reply:
It's probably fine: these DOI go into effect closer to the conference, which, in this case of OOPSLA'24, happens in the late October.

Reply:
ok cool, thanks!